src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -193,60 +194,29 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
     folderOnTrashLength,
     isTrash: isTrash,
     filesOnTrashLength,
-    setHasMoreItems,
+    setHasMoreItems: paginationState.setHasMoreItems,
     order,
   });
 
-  // RIGHT CLICK MENU STATES
-  const [isListElementsHovered, setIsListElementsHovered] = useState<boolean>(false);
-  const menuButtonRef = useRef<HTMLButtonElement>(null);
-  const [posX, setPosX] = useState(0);
-  const [posY, setPosY] = useState(0);
-  const [openedWithRightClick, setOpenedWithRightClick] = useState(false);
-  const [dimensions, setDimensions] = useState({ width: 0, height: 0 });
-  const containerRef = useRef<HTMLDivElement>(null);
-
-  // ONBOARDING TUTORIAL STATES
-  const [hasAnyUploadedFile, setHasAnyUploadedFile] = useState<boolean | undefined>();
-  const [currentTutorialStep, setCurrentTutorialStep] = useState(0);
-  const [showSecondTutorialStep, setShowSecondTutorialStep] = useState(false);
-  const uploadFileButtonRef = useRef(null);
-  const successNotifications = useTaskManagerGetNotifications({
-    status: [TaskStatus.Success],
-  });
-  const divRef = useRef<HTMLDivElement | null>(null);
-  const hasSignedToday = useAppSelector(userSelectors.hasSignedToday);
-
-  const showTutorial =
-    hasAnyUploadedFile !== undefined &&
-    !hasAnyUploadedFile &&
-    envService.isProduction() &&
-    hasSignedToday &&
-    (showSecondTutorialStep || currentTutorialStep === 0);
+  const showTutorial = tutorialState.showTutorial;
```

Comment:
There are any reason to maintain this const instead of use directly `tutorialState.showTutorial` in the Tutorial prop?